pkgs/by-name/pu/pushup/package.nix

```
@@ -0,0 +1,34 @@
+{ fetchFromGitHub, buildGoModule, lib, pkgs }:
+buildGoModule rec {
+  pname = "pushup";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "adhocteam";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-9ENXeVON2/Bt8oXnyVw+Vl0bPVPP7iFSyhxwc091ZIs=";
+  };
+
+  vendorHash = null;
+  subPackages = ".";
+  CGO_ENABLED = 0;
+  ldflags = [ "-s" "-w" ];
+  nativeBuildInputs = with pkgs; [ makeWrapper ];
+  allowGoReference = true;
+  postInstall = ''
+    wrapProgram $out/bin/${pname} --prefix PATH : ${
+      lib.makeBinPath (with pkgs; [ go ])
+    }
+  '';
+
+  meta = with lib; {
+    description = "Pushup web framework for Go";
```

```suggestion
    description = "A web framework for Go";
```
We don't want the package name at the start of the description.